zcash_primitives/src/transaction/components/sapling.rs

```diff
@@ -117,7 +211,7 @@ pub struct OutputDescription {
     pub ephemeral_key: jubjub::ExtendedPoint,
     pub enc_ciphertext: [u8; 580],
     pub out_ciphertext: [u8; 80],
-    pub zkproof: [u8; GROTH_PROOF_SIZE],
+    pub zkproof: GrothProofBytes,
```

Comment:
This also needs to be typed, since `OutputDescription` can be in an unproven state similar to `SpendDescription`.